src/home/WalletHome.tsx

```diff
@@ -184,7 +186,15 @@ function WalletHome() {
       style={styles.container}
       edges={!showHomeNavBar ? ['top'] : undefined}
     >
-      <DrawerTopBar middleElement={<Logo />} scrollPosition={scrollPosition} />
+      <DrawerTopBar
+        middleElement={showQrScanner ? undefined : <Logo testID="WalletHome/Logo" />}
```

Comment:
optional, should we use a separate parameter for this? although I guess this param, showHomeActions and showQrScanner are either all together true or false right now, so maybe overkill. The original intent was to not show valora logo for home actions, so maybe showHomeActions is more appropriate if we think a new param is overkill?